docs/cheatsheet.rst

```diff
@@ -139,40 +139,50 @@ The steps are as follows:
 
 This is where the actual magic happens.
 
-4. Commit it
+4. Add it
+
+-   add a single file
+::
+    hub add  filename
+
+-   add all changed files
+::
+    hub add  .
+
+5. Commit it
 ::
 
     hub commit -a -m "hey look, real work!"
```

Comment:
The `-a` flag automatically adds all of the changes that have been made, so it would be best to remove it here.

Reply:
nice catch